cmd/controller/main.go

```diff
@@ -36,9 +36,8 @@ const (
 var (
 	entrypointImage = flag.String("entrypoint-image", "override-with-entrypoint:latest",
 		"The container image containing our entrypoint binary.")
-	nopImage               = flag.String("nop-image", "tianon/true", "The container image used to stop sidecars")
-	affinityAssistantImage = flag.String("affinity-assistant-image", "nginx", "The container image used for the Affinity Assistant")
-	gitImage               = flag.String("git-image", "override-with-git:latest",
+	nopImage = flag.String("nop-image", "override-with-nop:latest", "The container image used to stop sidecars")
```

Comment:
Does nop image *runs* indefinitely ?

Reply:
The image runs in two modes, one which exits immediately (to stop sidecars) and another that runs indefinitely (to support Affinity Assistant). So it _can_ run indefinitely, but for sidecar-stopping it won't, and will just exit immediately.

Reply:
ohhhhhhhhh I didn't know about that :upside_down_face: 
Sounds very good then :stuck_out_tongue_closed_eyes: